### DISCUSSION TITLE
.2 release for EOS7?

### DISCUSSION THREAD

**Discussion by Eostux** [elementary/triage#172] [Q&A] [answered]
The .2 release of ubuntu will be released on Feb 23, 2023. At some point, EOS will be upgraded to the .2 release. Will the upgrade be available via updates or will it require a fresh install in order to utilize the .2 release of EOS when released?

**Comment by tintou** [comment, marked as answer]
As Ubuntu itself, doing regular updates will bring you to the .2 release :slightly_smiling_face:
